test/envs/error.yaml

```diff
@@ -12,5 +12,7 @@ profiles:
     workers:
       - addons:
           - name: error
+            args: ["10"]
```

Comment:
Need to add more addons to test that we don't continue to run addons after the failing addon.